### PR TITLE
[KOGITO-5104] - [1.5.x] - RHPAM Kogito operator creates wrong default image tags

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Many thanks for submitting your Pull Request :heart:! 

Jira issue : https://issues.redhat.com/browse/KOGITO-5104

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster
- [x] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: core/infrastructure/image.go [hunk 1]
```diff
@@ -193,14 +192,10 @@ func (i *imageHandler) ResolveImageStreamTriggerAnnotation(containerName string)
 
 // GetKogitoImageVersion gets the Kogito Runtime latest micro version based on the Operator current version
```
Comment (radtriste) [inline core/infrastructure/image.go:193]:
```suggestion
// GetKogitoImageVersion gets the Kogito Runtime latest micro version based on the given version
```

**Review by radtriste — CHANGES_REQUESTED** [review]
code looks good. one small change

**Comment by radtriste** [comment]
for the OLM test I guess the `replaces` should be set to `1.6.0` ?
or we would need to create a new channel ?

**Review by sutaakar — APPROVED** [review]

**Comment by sutaakar** [comment]
@vaibhavjainwiz Please open PR for master branch first, once merged there it can be backported.
Also please reference branch name in PR subject (if it is not master), otherwise it is easy to overlook that the PR doesn't target master.

**Comment by ricardozanini** [comment]
Haven't we closed a cherry-pick to 1.6.x? Why open one for 1.5.x? It seems that OLM is failing after this change.

**Comment by radtriste** [comment]
OLM is not related to the PR but a more global problem in the scripts.... @msmagnanijr is looking into it AFAIK
cherry-pick to 1.6.x was not useful because we won't release from this branch anymore. 1.5.x is still used for the product, so we need the change there ;)

**Review by radtriste — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**